std/encoding.d

```diff
@@ -2462,7 +2462,7 @@ abstract class EncodingScheme
      * Params:
      *    s = the array to be tested
      */
-    bool isValid(const(ubyte)[] s)
+    bool isValid()(const(ubyte)[] s)
```

Comment:
Looks like you made a couple of `public` functions templated.
Technically, this is a breaking change, since one those are not final and cannot be overriden anymore.
I let it to someone more informed about Phobos practice than me to decide whether this is okay.